### ISSUE TITLE
stats.slabserver.org not updated to 1.19

### ISSUE THREAD

**Issue by chrissamael** [DaMarine/Slabserver-Bugs#52]
### Server

Survival

### Describe the bug

The website is not updated to 1.19

### Expected behaviour

Seeing the new stars

### Steps to reproduce

Visit https://stats.slabserver.org
Look for 1.19 stats
Be sad 

### Checklist

- [X] I have checked the Slabserver-Bugs repo to make sure this is not reported, or marked as "Won't Fix".

**Comment by chrissamael** [comment]
This is blocked by [this issue](https://github.com/pdinklag/MinecraftStats/issues/197)

**Comment by DaMarine** [comment]
Commented on https://github.com/pdinklag/MinecraftStats/issues/197 that I believe the work for this is done now. Ran a `git pull` to bring in the latest Minecraft Stats version, and added it to our cronjob. 

Worth noting that the goat horn stat is bugged due to https://bugs.mojang.com/browse/MC-250943, we'll likely get a question or two about this.
